external/esp_idf_port/esp32/esp32_wifi_os_adapter.c

```diff
@@ -630,7 +630,7 @@ uint32_t IRAM_ATTR esp_get_free_heap_size(void)
 
 static void *IRAM_ATTR malloc_internal_wrapper(size_t size)
 {
-#if (defined(CONFIG_SPIRAM_SUPPORT) && CONFIG_MM_REGIONS > 1)
+#if (defined(CONFIG_SPIRAM_SUPPORT) && CONFIG_MM_NHEAPS > 1)
 	return malloc_at(0, size);
```

Comment:
look at malloc_at defination, it is controlled by CONFIG_MM_NHEAPS  itself.
so for code conciseness,  suggest moving this macro when calling it.

Reply:
In this case: CONFIG_SPIRAM_SUPPORT is defined, but CONFIG_MM_NHEAPSis not defined.  As your method, it will cause compling issue. Please check it.